common/python/utility/utility.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+utility.py -- general helper function
+"""
+import base64
+import os
+import utility.file_utils as putils
+import utility.hex_utils as hex_utils
+import crypto_utils.crypto.crypto as crypto
+import config.config as pconfig
+import logging
+
+logger = logging.getLogger(__name__)
+
+def create_error_response(code, jrpc_id, message):
```

Comment:
This should be moved to a different error handling file. Or create a json_utils.py

Reply:
I removed all the unused functions in utility.py and renamed this file to jrpc_utils.py as the file now has only one jrpc utility function and nothing else.